widget/public/index.html

```diff
@@ -26,7 +26,13 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="ab_wpr_login_button_root"></div>
```

Comment:
Can you confirm that removing/updating this line won't break another clients implementation? I'm afraid if you remove/change this id, we will need to adjust the implementation on another client as well

Reply:
I don't think we're using this html file in the WordPress, since the HTML template is the WordPress page itself